flexmeasures/data/models/planning/utils.py

```diff
@@ -208,6 +209,8 @@ def get_power_values(
         )
         df = df.fillna(0)
 
+    df *= convert_units(1, sensor.unit, to_unit)
```

Comment:
I think if you pass `df` instead of `1` this method may even allow converting from kWh to kW using the sensor's resolution.